### DISCUSSION TITLE
How can I manually set the 2D-positions of the non-C atoms of a molecule?

### DISCUSSION THREAD

**Discussion by wikpron** [rdkit/rdkit#4048] [Q&A]
How can I perform the following steps using rdkit in python
1. manually set the 2d-positions of non-hydrogen atoms in 2D
2. set the bond directions wedge/dash to subsequently compute atom chirality

**Comment by greglandrum** [comment]
Ha... I coincidentally just provided an answer to this question here: https://github.com/rdkit/rdkit/discussions/4036#discussioncomment-620936
